client/my-sites/theme/main.jsx

```diff
@@ -661,21 +643,27 @@ class ThemeSheet extends Component {
 						<span className="theme__sheet-main-info-tag">{ tag }</span>
 					</div>
 					<div className="theme__sheet-main-actions">
-						{ config.isEnabled( 'themes/block-theme-previews-poc' ) && (
+						{ shouldRenderButton &&
+							( this.shouldRenderUnlockStyleButton()
+								? this.renderUnlockStyleButton()
+								: this.renderButton() ) }
+						{ config.isEnabled( 'themes/block-theme-previews' ) && (
 							<LivePreviewButton
 								isActive={ isActive }
 								isAtomic={ isAtomic }
 								isExternallyManagedTheme={ isExternallyManagedTheme }
+								isFullSiteEditingTheme={ isFullSiteEditingTheme }
+								isSimple={ isSimple }
+								isThemeInstalledOnAtomicSite={ isThemeInstalledOnAtomicSite }
 								isWporg={ isWporg }
 								showTryAndCustomize={ showTryAndCustomize }
 								siteSlug={ siteSlug }
 								stylesheet={ stylesheet }
+								themeId={ themeId }
+								themeType={ themeType }
```

Comment:
Instead of `themeType`, we can check `isThemePurchased` or `canUseTheme`. Upgraded users should be able to Live Preview premium themes, for instance.

Reply:
Thanks for your comment! Addressed in https://github.com/Automattic/wp-calypso/pull/79775/commits/55d5ef7c7a32311e7863d10d4cc214777cb65fc0.